python/examples/music/client.py

```diff
@@ -0,0 +1,378 @@
+import os
+import sys
+
+current_path = os.path.abspath(os.path.dirname(__file__))
+if current_path not in sys.path:
+    sys.path.append(current_path)
+
+import math
+from typing import Any, Optional
+
+from pydantic.dataclasses import dataclass
+import spotipy
+
+from schema import PlayerAction
+from spotipyWrapper import SimplifiedTrackInfo, SimplifiedPlaylistInfo, AsyncSpotipy
+
+
+class Config:
+    arbitrary_types_allowed = True
+
+
+@dataclass(config=Config)
```

Comment:
Why do you need this to be a Pydantic dataclass? It doesn't look like you serialize it anywhere.

It looks like you just want a regular `dataclass` so you can get an `__init__` for free, right?